src/Icon/Icon.stories.tsx

```diff
@@ -0,0 +1,14 @@
+// stories/MyButton.stories.tsx
+import React from 'react'
+import type {ComponentMeta, ComponentStory} from '@storybook/react'
+
+import {Icon} from './Icon'
+
+export default {
+  title: 'components/MyButton',
+  component: Icon,
+} as ComponentMeta<typeof Icon>
+
+export const IconComponent: ComponentStory<typeof Icon> = args => (
+  <Icon source={require('../assets/flash-icon.png')} />
```

Comment:
This image don't exist

Reply:
done

Reply:
I think it's fine because this is a storybook to ensure that the button UI renders correctly
We don't need to add a local image to test this feature